client/src/components/Navbar.js

```diff
@@ -36,23 +37,33 @@ const SearchBar = styled.input`
 export const Navbar = () => {
   return (
     <nav className="nav">
-      <Title>Project Metroid</Title>
+      <Title className="title">Project Metroid</Title>
+      <button className="mobile-search-toggle nav__mobile--link">
```

Comment:
A couple of things here.  Let's avoid prefixing classes with `mobile`.  Rather than that we should call it something more meaning full and just toggle the mobile styles in our media queries.  For something like this it would be more clear to name it something like `button button--search`.  Then we need to decide what this really is, is it a link that routes you to a new page? Or is it a button that toggles some action?  The two classes are contradicting.